app/controllers/api/v1/customer_controller.rb

```diff
@@ -0,0 +1,40 @@
+class Api::V1::CustomerController < Api::V1::ApiController
+
+  def index
+    @customers = Customer.all.where(company: @company)
```

Comment:
Como você está herdando de `Api::V1::ApiController` é necessário que o "requisitor" esteja vinculado a uma `Company`.

Teoricamente, seria possível alterar a query para realizá-la utilizando essa referência? 🤔 

Algo como:
```suggestion
    @customers = @company.customers
```